src/common/contentPreference.ts

```diff
@@ -107,7 +113,7 @@ export const cleanContentNotificationPreference = async ({
   });
 };
 
-const followUser: FollowEntity = async ({ ctx, id, status }) => {
+const followUser: FollowEntity = async ({ ctx, id, status, feedId }) => {
```

Comment:
Can we add a test to make sure additions work for mutations?

Reply:
Yeah, I'll add some tests. Was holding off a bit on that to make sure I hadn't completely missed the objective of this task